crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -1012,7 +1022,7 @@ impl From<NtStatus> for u32 {
 /// [2.2.1.4 Device I/O Request (DR_DEVICE_IOREQUEST)]
 ///
 /// [2.2.1.4 Device I/O Request (DR_DEVICE_IOREQUEST)]: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/a087ffa8-d0d5-4874-ac7b-0494f63e2d5d
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
Is derivating the `Copy` trait a deliberate decision?
Since this type is a POD, not _too_ big (20 bytes), it’s not unreasonable at all though I don’t think we did that on the others (more of an API-wise consideration than a performance-wise one; fwiw, maybe we should implement `Copy` on more types at some point).

Reply:
This was added so that I could pass a `DeviceIoRequest` without taking ownership here

https://github.com/gravitational/teleport/blob/bcb7125512155f9036401181a02efa380e90e385/lib/srv/desktop/rdp/rdpclient/src/rdpdr/filesystem.rs#L399-L408

however I didn't give any thought to the implications for the API. Having thought about it more, I think we can be conservative with respect to deriving `Copy` on full PDUs (I'll remove it here), and revisit it again later if I find myself bumping into it commonly over in Teleport.

Reply:
Sounds good, keep me informed!